client/src/fabric/FabricRuntime.ts

```diff
@@ -77,6 +82,81 @@ export class FabricRuntime extends EventEmitter {
         return this.state;
     }
 
+    public async create(): Promise<void> {
+
+        // Delete any existing runtime directory, and then recreate it.
+        await fs.remove(this.path);
```

Comment:
remember yesterday when I had to delete the runtime folder to get stuff to work? shouldn't this code have been run yesterday?

Reply:
@heatherlp it did run, but the generator didn't work (as you hadn't run npm install) so you were left with an empty runtime folder. The check for creation is "does the runtime folder" exist, so it didn't recover from the problem.